### DISCUSSION TITLE
How to add ingredients?

### DISCUSSION THREAD

**Discussion by Rickman** [mealie-recipes/mealie#542] [Q&A]
I might be missing something here, but I don't seem to have the option to add ingredients.

![mealie_ss](https://user-images.githubusercontent.com/1157647/122411015-54c4a580-cf7c-11eb-9b73-8b1370319595.PNG)

 All the guides I've seen has an ingredients section already there when creating a new recipe.

What am I doing wrong? Surely this is one of the most important sections when creating a recipe!

**Comment by sephrat** [comment]
That's a bug: https://github.com/hay-kot/mealie/issues/538

**Reply by Rickman** [reply]
Ah right, that's a bugger, thanks for the quick reply!

**Reply by wengtad** [reply]
and it fixed in v0.5.1, you may pull and update
